### ISSUE TITLE
Empty negotiated response with accept header of application/xml or text/xml throws serialization exception

### ISSUE THREAD

**Issue by phoenixx** [NancyFx/Nancy#2131]
Given a simple model which simply returns a 200 response:

```c#
 public class DemoModule : NancyModule
    {
        public DemoModule() : base("/demo")
        {
            Get["ok", "/"] =
                _ => Negotiate.WithStatusCode(HttpStatusCode.OK);
        }
    }
```

Making a GET request to this endpoint with an accept header of "application/json" returns an empty response with a status code of 200, as expected. 

Making a GET request to this endpoint with an accept header of "text/xml" or "application/xml" returns a 500 internal server error with the following exception:

````
The type arguments for method 'Nancy.ISerializer.Serialize<TModel>(string, TModel, System.IO.Stream)' cannot be inferred from the usage. Try specifying the type arguments explicitly.
````

**Comment by khellang** [comment]
Closed by #2133

**Comment by khellang** [comment]
Version 1.4.3 was just pushed to [NuGet](https://www.nuget.org/packages/Nancy/1.4.3).
